rust/telemetry/src/lib.rs

```diff
@@ -68,7 +68,11 @@ impl Telemetry {
                 ..Default::default()
             },
         ));
-        sentry::configure_scope(|scope| scope.set_tag("api_url", api_url));
+        // Configure scope on the main hub so that all threads will get the tags
+        sentry::Hub::main().configure_scope(|scope| {
+            scope.set_tag("api_url", api_url);
+            scope.set_tag("os", std::env::consts::OS);
```

Comment:
Should we use the `OsContext` here? https://docs.rs/sentry/latest/sentry/struct.Scope.html#method.set_context

Reply:
Oh yeah, turns out we can make Sentry gather a bunch of that for us
<img width="816" alt="image" src="https://github.com/user-attachments/assets/9520a0ca-24b4-4d6a-825e-d2dd2ec5611a">

No distro name though :| I'll pull it from `/etc/issue` or something eventually